### PR TITLE
merge the Chinese story branch into the master branch

### PR DESCRIPTION
**Pull request by MakerRuby**
I have merged the master branch of the original repository into my branch.@casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: Chinese/大话西游.md [hunk 1]
```diff
@@ -5,3 +5,6 @@
 [Casey](Casey/Casey.md) ?
 
 [YW](YW/yw.md) ?
+
+
+[警察](警察/警察.md) ?
```
Comment (casey-collab) [inline Chinese/大话西游.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Hm, I'm not sure why @casey-collab thinks the link is broken. Let me debug that.

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
@casey-collab, please check again

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Okay, sorry for all the comment spam. Everything checks out now, except that the merge conflicts need to be resolved.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to resolve the merge conflicts, feel free to re-open it.

**Pull request closed without merging**